azure-storage-spring-boot-autoconfigure/src/main/java/com/microsoft/azure/autoconfigure/azurestorage/StorageProperties.java

```diff
@@ -9,7 +9,7 @@
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
 @ConfigurationProperties("azure.storage")
-public class AzureStorageProperties {
+public class StorageProperties {
 
     private String connectionString;
```

Comment:
Instead of one string it would be better to add three values like DefaultEndpointsProtocol , AccountName and AccountKey and form connectionstring in java.

Any suggestions is welcome. If you are ok with this I will submit PR